docs/modules/ROOT/pages/release-notes.adoc

```diff
@@ -1,28 +1,33 @@
 
 == Changes
 
-- Added the ability to configure Interfaces for ClientServer (https://github.com/hazelcast/hazelcast-platform-operator/pull/920[#920])
-- The new `hazelcastendpoint` Custom Resource Definition (CRD) provides a better way to list the external addresses of your Hazelcast cluster (https://github.com/hazelcast/hazelcast-platform-operator/pull/860[#860])
-- A `hazelcast-resource` column has been added to the `additionalPrinterColumns` for the wide output of CRDs referencing Hazelcast CR (https://github.com/hazelcast/hazelcast-platform-operator/pull/899[#899])
-- Failed HotBackup upload to bucket now correctly handles partially uploaded files and tries to remove them (https://github.com/hazelcast/hazelcast-platform-operator/pull/885[#885])
-- The `v.Err` function was moved to `fieldValidator` to remove duplication (https://github.com/hazelcast/hazelcast-platform-operator/pull/886[#886])
-- Improved validations for HotBackup, WANReplication, JetJob, JetJobSnapshot, Map, Cache, and Topic (https://github.com/hazelcast/hazelcast-platform-operator/pull/877[#877])(https://github.com/hazelcast/hazelcast-platform-operator/pull/876[#876])(https://github.com/hazelcast/hazelcast-platform-operator/pull/875[#875])(https://github.com/hazelcast/hazelcast-platform-operator/pull/874[#874])(https://github.com/hazelcast/hazelcast-platform-operator/pull/873[#873])(https://github.com/hazelcast/hazelcast-platform-operator/pull/872[#872])(https://github.com/hazelcast/hazelcast-platform-operator/pull/871[#871])(https://github.com/hazelcast/hazelcast-platform-operator/pull/859[#859])
+- Removed the ability to configure `persistence.baseDir` field for user convenience (https://github.com/hazelcast/hazelcast-platform-operator/pull/1028[#1028])
+- Added support for running Hazelcast cluster with user-defined `serviceAccount` (https://github.com/hazelcast/hazelcast-platform-operator/pull/983[#983])
+- Display HazelcastEndpoint address for NodePort services (https://github.com/hazelcast/hazelcast-platform-operator/pull/966[#966])
+- Property `hazelcast.cluster.version.auto.upgrade.enabled` was enabled by default (https://github.com/hazelcast/hazelcast-platform-operator/pull/957[#957])
+- Support for scaling the cluster size using the `scale` subresource has been added (https://github.com/hazelcast/hazelcast-platform-operator/pull/941[#941])
+- Users are now able to define annotations and labels for Hazelcast resources, which Operator will use when creating new Kubernetes resources (https://github.com/hazelcast/hazelcast-platform-operator/pull/964[#964])
+- Added ‘delete’ Webhook to HotBackup Custom Resource Definition (CRD)  for checking to restore references (https://github.com/hazelcast/hazelcast-platform-operator/pull/981[#981])
+- WanConfig name field length limit was set to prevent port name generation issues (https://github.com/hazelcast/hazelcast-platform-operator/pull/936[#936])
```

Comment:
```suggestion
- Set the WanConfig name field length limit to prevent port name generation issues (https://github.com/hazelcast/hazelcast-platform-operator/pull/936[#936])
```